hub/package-manager/winget/install.md

```diff
@@ -51,9 +52,11 @@ The options allow you to customize the install experience to meet your needs.
 | **--override** | A string that will be passed directly to the installer.    |
 | **-l, --location** |    Location to install to (if supported). |
 | **--force** | Overrides the installer hash check. Not recommended. |
+| **--dependency-source** | Find package dependencies using the specified source. |
```

Comment:
Should this include an example? 
--dependency-source https://www.contoso.com/cache

(not sure if this is correct usage)

Reply:
We haven't implemented any support for --dependency-source, so I agree we should either remove it, or explicitly call out that it is not yet supported in the client.

Reply:
okay, I did not know that we don't have support for that yet. I removed it for now and will add it back when it is supported